checker/src/main/java/org/checkerframework/checker/determinism/DeterminismVisitor.java

```diff
@@ -525,6 +526,59 @@ public Void visitMethod(MethodTree node, Void p) {
         return super.visitMethod(node, p);
     }
 
+    /**
+     * If the declaration of {@code node} is annotated with {@code @RequiresDetToString}, checks
```

Comment:
This Javadoc doesn't match the implementation. 
1. It's both a param type of Object and a param type of Object[].
2. The toString can return either a `@Det` or `@PolyDet`.